third_party/2and3/cryptography/x509.pyi

```diff
@@ -277,15 +278,17 @@ class Extensions(object):
     def __init__(self, general_names: List[Extension]) -> None: ...
     def __iter__(self) -> Generator[Extension, None, None]: ...
     def get_extension_for_oid(self, oid: ObjectIdentifier) -> Extension: ...
-    def get_extension_for_class(self, extclass: ExtensionType) -> Extension: ...
+    def get_extension_for_class(self, extclass: Type[ExtensionType]) -> Extension: ...
 
 class IssuerAlternativeName(ExtensionType):
     def __init__(self, general_names: List[GeneralName]) -> None: ...
     def __iter__(self) -> Generator[GeneralName, None, None]: ...
+    def get_values_for_type(self, type: Type[GeneralName]) -> Generator[GeneralName, None, None]: ...
```

Comment:
As far as I can see in the implementation, this (and the one below) just delegate to `GeneralNames.get_values_for_type()`, which just returns a list of the values from the GeneralNames. So shouldn't the return value be `List[Any]`?

Reply:
Good eye!  I was so focused on the arguments, I didn't scrutinize the return. 

It is a list, and you are correct, only another Any will do :/ .   It could be Union[Any, OtherName] but that seems superfluous.   updated patch (squashed for not polluting the history)